SR2020/2019-08-31-announce-SR2020.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Previously registered interested team leaders
+subject: SR2020 Registration and Kickstart Announcement
+---
+
+Hi,
+
+We're pleased to announce that registration for Student Robotics 2020 is now
+open. If you would like to compete, please complete this [application form][signup-form]
+with the details of your team. All future emails about SR2020 will be sent there.
+
+This year we're hosting three Kickstart events, all on **TODO DATE**:
```

Comment:
Maybe just define a ballpark date if we don't have one by the time this email should go out.

Reply:
cfbd8fa